source/views/streaming/radio/index.js

```diff
@@ -18,10 +18,12 @@ import * as c from '../../components/colors'
 import Icon from 'react-native-vector-icons/Ionicons'
 import {Touchable} from '../../components/touchable'
 import {TabBarIcon} from '../../components/tabbar-icon'
+import {phonecall} from 'react-native-communications'
 import type {TopLevelViewPropsType} from '../../types'
 
 const kstoStream = 'https://cdn.stobcm.com/radio/ksto1.stream/master.m3u8'
 const image = require('../../../../images/streaming/ksto/ksto-logo.png')
+const stationNumber = '5077863602'
```

Comment:
Please change to `+15077863602`. (Add prefix `+1` country code for international users.)  Doubt it'll actually make a difference for these users due to system-wide settings, but it's nice to have.